File: Docker/docker-auth-MERN/backend/src/utils/env.js
import dotenv from "dotenv";
dotenv.config();

export const {
  DB_URI,
  JWT_SECRET,
  JWT_ALGORIHM,
  JWT_EXPIRES_IN,
  CLOUDINARY_NAM,
  CLOUDINARY_SECRT,
  CLOUDINARY_API_KE,
  FRONT_END_ORIGIN,
  PORT,
} = process.env;

File: Projects/auth-fast-mern/fast-backend/app/utils/env.py
from dotenv import load_dotenv
import os

load_dotenv()

DB_URI= os.getenv("DB_URI")
JWT_SECRET= os.getenv("JWT_SECRET")
JWT_ALGORITHM= os.getenv("JWT_ALGORITHM")
JWT_EXPIRES_IN= os.getenv("JWT_EXPIRES_IN")
